Source/ARTMessage.h

```diff
@@ -16,4 +17,11 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+@interface ARTMessage (Decryption)
+
++ (instancetype)fromEncodedJsonObject:(NSDictionary *)json channelOptions:(ARTChannelOptions *)options error:(NSError **)error;
```

Comment:
I wonder if return value shouldn't be `nullable`?

Reply:
It's because `ARTBaseMessage/decodeWithEncoder` doesn't do. I think this change should be in another PR.

Reply:
Made it here anyway: https://github.com/ably/ably-cocoa/pull/1307/commits/3b58d439a6a692b4874e4b12f22c6ea3c8778b74 https://github.com/ably/ably-cocoa/pull/1307/commits/6fb0756453292dc06ff292a2e413eac9929f69b8